tests/test_ascii_art.py

```
@@ -0,0 +1,61 @@
+        ]
+        expected_ascii_art = " # \n###\n # \n"
+        ascii_art = convert_pixel_art_to_ascii(pixel_art)
+        self.assertEqual(ascii_art, expected_ascii_art)
+
+    def test_load_ascii_art(self):
+        filename = "test_art.txt"
+        with open(filename, "w") as file:
+            file.write("Test ASCII Art")
+        loaded_art = load_ascii_art(filename)
+        self.assertEqual(loaded_art, "Test ASCII Art")
+        os.remove(filename)
```

Ruff (0.8.0)

Undefined name `os`

(F821)